nservicebus/operations/opentelemetry_content_core_[8,).partial.md

```diff
@@ -21,6 +21,12 @@ snippet: opentelemetry-enablemeters
 
 See the [OpenTelemetry samples](/samples/open-telemetry/) for instructions on how to send metric information to different tools.
```

Comment:
Maybe we can move this section down below the "Emitted meters" section as a final CTA for the Meters section?

Reply:
Agreed, I have moved it